### PR TITLE
Removed preguarding of the SDKs.

### PR DESCRIPTION
**Pull request by bjornick**
Fixes https://github.com/firebase/firebase-android-sdk/issues/45

### CODE REVIEW

File: root-project.gradle [hunk 1]
```diff
@@ -126,7 +126,7 @@ configure(subprojects) {
         // Apply the retrolambda plugin
         sub.apply plugin: 'me.tatarka.retrolambda'
 
-        boolean skipPreguard = sub.getProperties().getOrDefault('firebaseSkipPreguard', 'false').toBoolean()
+        boolean skipPreguard = sub.getProperties().getOrDefault('firebaseSkipPreguard', 'true').toBoolean()
```
Comment (ashwinraghav) [inline root-project.gradle]: too easy 🤣

**Comment by ashwinraghav** [comment]
I think it is in scope to 
1) Remove the @~PublicApi annotations for the projects. We should retain @~Hide and @~KeepForSdk annotations. Right?
2) Remove the preguard files for the projects

**Comment by bjornick** [comment]
We want to keep the @PublicApi annotation around.  If nothing else, it documents in our repo clearly what our supported API is and what is subject to our semver versions.  The added tooling is a bonus.

Remove the preguard.txt files. PTAL

**Pull request merged**